packages/components/src/dropdown-menu/index.js

```diff
@@ -129,7 +129,7 @@ function DropdownMenu( {
 						aria-haspopup="true"
 						aria-expanded={ isOpen }
 						label={ label }
-						showTooltip
+						showTooltip={ toggleProps?.showTooltip || true }
```

Comment:
This will always set `showTooltip` regardless of what `toggleProps?.showTooltip` is.

Assuming that `true` is supposed to be a default value that can be overridden by `toggleProps?.showTooltip`, we could use `??` instead of `||`:

```suggestion
						showTooltip={ toggleProps?.showTooltip ?? true }
```

Reply:
Oooh, well spotted! Fixing.